docs/docs/ecosystem/community.md

```diff
@@ -8,61 +8,111 @@ Sponsors, contributors, and adopters make this ecosystem possible and shape it
 in significant ways.
 
 Important community resources include [GitHub](https://github.com/Ory),
-[Slack (chat)](https://slack.Ory.sh/) and GitHub Discussions for
+[Ory Chat](https://slack.Ory.sh/) and GitHub Discussions for
 [Ory Kratos](https://github.com/ory/kratos/discussions),
 [Ory Hydra](https://github.com/ory/hydra/discussions),
 [Ory Keto](https://github.com/ory/oathkeeper/discussions),
 [Ory Oathkeeper](https://github.com/ory/oathkeeper/discussions) and all other
 [Ory projects](https://github.com/ory/meta/discussions), you can also visit our
-archived [forum](https://community.Ory.sh/).
+archived old [forum](https://community.Ory.sh/).
```

Comment:
Looks like all `ory` got replaced with `Ory` introduced by [this commit](https://github.com/ory/docs/commit/b884b68a8bfc86e6d9f2746eed78ca8cb28df67b). IntelliJ has a setting which respects casing. It's also a good idea to make the replacements case sensitive.

Can you please create another PR where the domain names are properly cased?

Reply:
Sure, my bad!